platform/solr/solr-schema/src/main/resources/solr/conf/solrconfig.xml

```diff
@@ -870,47 +833,41 @@
 
-    <str name="queryAnalyzerFieldType">text_general</str>
-
-    <!-- Multiple "Spell Checkers" can be declared and used by this
-         component
-      -->
+    <str name="queryAnalyzerFieldType">text_suggest</str>
 
-    <!-- a spellchecker built from a field of the main index -->
     <lst name="spellchecker">
-      <str name="name">default</str>
-      <str name="field">_text_</str>
-      <str name="classname">solr.DirectSolrSpellChecker</str>
-      <!-- the spellcheck distance measure used, the default is the internal levenshtein -->
-      <str name="distanceMeasure">internal</str>
-      <!-- minimum accuracy needed to be considered a valid spellcheck suggestion -->
-      <float name="accuracy">0.5</float>
-      <!-- the maximum #edits we consider when enumerating terms: can be 1 or 2 -->
-      <int name="maxEdits">2</int>
-      <!-- the minimum shared prefix when enumerating terms -->
+      <!-- The spellcheck index will need to be built once for this dictionary before it will work -->
+      <str name="name">filebased</str>
+      <str name="classname">solr.FileBasedSpellChecker</str>
+      <str name="sourceLocation">dictionary.txt</str>
+      <str name="spellcheckIndexDir">./spellcheckerFile</str>
+      <str name="characterEncoding">UTF-8</str>
+    </lst>
+    <lst name="spellchecker">
+      <str name="name">indexbased</str>
+      <str name="field">spellcheckfield</str>
+      <str name="classname">solr.IndexBasedSpellChecker</str>
+      <str name="distanceMeasure">org.apache.lucene.search.spell.LevenshteinDistance</str>
+      <str name="accuracy">0.5</str>
+      <int name="maxEdits">1</int>
       <int name="minPrefix">1</int>
-      <!-- maximum number of inspections per result. -->
       <int name="maxInspections">5</int>
-      <!-- minimum length of a query term to be considered for correction -->
       <int name="minQueryLength">4</int>
-      <!-- maximum threshold of documents a query term can appear to be considered for correction -->
-      <float name="maxQueryFrequency">0.01</float>
-      <!-- uncomment this to require suggestions to occur in 1% of the documents
-        <float name="thresholdTokenFrequency">.01</float>
-      -->
+      <str name="maxQueryFrequency">0.01</str>
+      <str name="buildOnCommit">true</str>
     </lst>
-
-    <!-- a spellchecker that can break or combine words.  See "/spell" handler below for usage -->
-    <!--
     <lst name="spellchecker">
       <str name="name">wordbreak</str>
       <str name="classname">solr.WordBreakSolrSpellChecker</str>
-      <str name="field">name</str>
+      <str name="field">spellcheckfield</str>
+      <!-- should multiple words from the query be combined in a dictionary search -->
       <str name="combineWords">true</str>
+      <!-- should words from the query be broken in a dictionary search -->
       <str name="breakWords">true</str>
-      <int name="maxChanges">10</int>
+      <int name="maxChanges">5</int>
```

Comment:
❓ Why was this value changed?

Reply:
maxChanges is an integer which defines how many times the spell checker should check collation possibilities against the index (default is 10). We thought 5 would be a better number and lest costly since we sort the collations in the backend depending on the number of hits in each collated search.